packages/react-components/react-tags/src/components/TagGroup/TagGroup.types.ts

```diff
@@ -10,14 +11,27 @@ export type TagGroupSlots = {
   root: Slot<'div'>;
 };
 
+export type TagGroupDismissData = {
+  dismissedTagId: string;
```

Comment:
shouldn't this become `dismissedTagValue`?

Reply:
Oops yes

Reply:
renamed :)